File: TrialsByFire/ViewController.m
//
//  ViewController.m
//  TrialsByFire
//
//  Created by jpk on 6/1/15.
//  Copyright (c) 2015 Parker Kirby. All rights reserved.
//

#import "ViewController.h"

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];

    
    NSMutableString * name = [@"Parker" mutableCopy];
    NSString * oldname = name;
    
    [name insertString:@"Cool " atIndex:0];
    
    name = [@"Troll" mutableCopy];
    
    NSLog(@"%@", oldname);

}

- (void)didReceiveMemoryWarning {
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}

@end

File: Review/Review/ViewController.swift
//
//  ViewController.swift
//  Review
//
//  Created by jpk on 6/16/15.
//  Copyright (c) 2015 Parker Kirby. All rights reserved.
//

import UIKit

class ViewController: UIViewController {

    override func viewDidLoad() {
        super.viewDidLoad()

        //if user == nil
        
        let loginStoryboard = UIStoryboard(name: "Login", bundle: nil)
        let loginVC = loginStoryboard.instantiateInitialViewController() as! UIViewController
        
        presentViewController(loginVC, animated: true, completion: nil)
    
        
    
    }

    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
        // Dispose of any resources that can be recreated.
    }


}

